### ISSUE TITLE
broken design on login page

### ISSUE THREAD

**Issue by mkudlej** [Tendrl/dashboard#224]
There is broken design on login page, please see screenshot.
![bug_tendrl_login](https://cloud.githubusercontent.com/assets/5920442/24698689/d92bb98e-19f1-11e7-9093-702b616c3aec.png)

It seems that is is similar bug as in Skyring https://bugzilla.redhat.com/show_bug.cgi?id=1298535

**Comment by sankarshanmukhopadhyay** [comment]
Could you also add detail about the package NVRs used to set up the instance where you notice this?

**Comment by cloudbehl** [comment]
@mkudlej can you please pass browser details also like-
On which browser you are testing and what is the version of that browser? That will be helpful.

**Comment by mkudlej** [comment]
Sorry about that:
```
etcd-3.1.0-2.el7.x86_64
python-etcd-0.4.3-1.noarch
rubygem-etcd-0.3.0-1.el7.centos.noarch
tendrl-api-1.2.1-04_05_2017_01_51_04.noarch
tendrl-api-doc-1.2.1-04_05_2017_01_51_04.noarch
tendrl-api-httpd-1.2.1-04_05_2017_01_51_04.noarch
tendrl-commons-1.2.2-04_05_2017_04_42_03.noarch
tendrl-dashboard-1.2.1-04_05_2017_11_03_12.noarch
tendrl-node-agent-1.2.2-04_05_2017_00_01_04.noarch
tendrl-performance-monitoring-1.2.2-04_05_2017_13_08_12.noarch
```
and 
```
Mozilla/5.0 (X11; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0
```

**Comment by gnehapk** [comment]
@mkudlej Not able to reproduce this issue in our system. Can you please check with the new build that is available "tendrl-dashboard: 1.2.1-04_09_2017_18_25_13".

**Comment by nthomas-redhat** [comment]
Closing this issue as this is not reproducible. Please re-open if found again
